packages/ui/components/editor/Editor.tsx

```diff
@@ -83,8 +83,10 @@ export const Editor = (props: TextEditorProps) => {
             setFirstRender={props.setFirstRender}
           />
           <div
-            className={classNames("editor-inner scroll-bar", !editable && "!bg-subtle")}
-            style={{ height: props.height }}>
+            className={classNames(
+              "editor-inner scroll-bar !h-[10rem] !resize-none",
```

Comment:
We want the text-area to be resizable, also don't increase the height that much

```suggestion
              "editor-inner scroll-bar !h-[10rem]",
```

Reply:
the reason i think the text area shouldn't be resizable is it looks like normal html simple text area and not all then majority of the websites have text area of fixed size